docs/user_guide.md

```diff
@@ -831,13 +836,13 @@ CodeChecker server -w ~/codechecker_wp --dbname myProjectdb --dbport 5432 --dbad
 The checking process can be started separately on the same machine
 
 ~~~~~~~~~~~~~~~~~~~~~
-CodeChecker check  -w ~/codechecker_wp -n myProject -b "make -j 4" --host localhost --port 8001
+CodeChecker check  -w ~/codechecker_wp -n myProject -b "make -j 4" --url localhost:8001/Default
```

Comment:
In theory, you can emit the **entire** `--url` parameter, as it defaults to `localhost:8001/Default` automatically. However, I think it is better, if the very advanced usage guides explicitly state that this argument is there.

Just as how, before the change, `--host` and `--port` was there, even though it was *not* needed, as this example here set them to their default values already.